index.html

```diff
@@ -918,107 +918,11 @@ <h2 id="BaseAudioContext">
             </p>
           </dd>
           <dt>
-            Promise&lt;void&gt; suspend()
```

Comment:
1. The current suspend() method definition moves from `BaseAudioContext` to `AudioContext`. (removing)